schemaregistry/avro_generic.go

```diff
@@ -0,0 +1,112 @@
+package schemaregistry
```

Comment:
is there any overhead associated with having avro/proto/json all in the one package from a dependency perspective?

Reply:
if there is, i guess probably just compile time, and probably not significant?

Reply:
Yes I don't believe there is